Basic/DyanmicPrefabs/Assets/Scripts/01_Server authoritative dynamic prefab loading across peers/AddressableGUID.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Unity.Collections;
+using Unity.Netcode;
+
+namespace Game
+{
+    public struct AddressableGUID : INetworkSerializable
+    {
+        public FixedString128Bytes Value;
```

Comment:
```suggestion
        // could use a byte or a short if client and server has the same list in the same order (use the index in the list instead of the actualy 128 bits for a full GUID)
        public FixedString128Bytes Value;
```